### PR TITLE
change version of black in pre commit

### PR DESCRIPTION
**Pull request by mohawk811**
## Description

This changes the version of black to 23.1.0

### CODE REVIEW

**Comment by namurphy** [comment]
It looks like the "Add comment/checklist" check is unrelated, so I'll look into that later.

File: docs/about/credits.rst [hunk 1]
```diff
@@ -88,6 +88,7 @@ in parentheses are `ORCID author identifiers <https://orcid.org>`__.
 * :user:`Marco Gorelli <MarcoGorelli>`
 * :user:`Maria Isupova <misupova>`
 * Michael Fischer
+* :user:`Mohammed Haque <mohawk811>`
```
Comment (namurphy) [inline docs/about/credits.rst:91]: Awesome!

**Review by namurphy — APPROVED** [review]
LGTM and thank you!

**Review by StanczakDominik — APPROVED** [review]
LGTM! I heard the changes were relatively drastic, glad to see it wasn't that bad here. Thanks!

**Pull request merged**